lib/httplib/httplib_test.go

```diff
@@ -374,6 +375,67 @@ func TestSetIndexContentSecurityPolicy(t *testing.T) {
 			}
 		})
 	}
+
+	indexCSPStringCache = newCSPCache()
+}
+
+func TestSetIndexContentSecurityPolicy_OSSBuild(t *testing.T) {
+	modules.SetTestModules(t, &modules.TestModules{TestBuildType: modules.BuildOSS})
+
+	for _, tt := range []struct {
+		name            string
+		features        proto.Features
+		urlPath         string
+		expectedCspVals map[string]string
+		buildType       string
+	}{
+		{
+			name:     "default (no stripe or wasm)",
+			features: proto.Features{},
+			urlPath:  "/web/index.js",
+			expectedCspVals: map[string]string{
```

Comment:
Suggestion: create the "default" set of values and modify it for each test, so instead of a large map in each scenario we can instead how it changes compared to the default.